src/Inciter/Partitioner.cpp

```diff
@@ -100,10 +105,9 @@ Partitioner::Partitioner(
   // only
   ownBndNodes( m_lid, m_bnode );
 
-  // Compute number of cells across whole problem
-  std::size_t nelem = m_ginpoel.size()/4;
-  contribute( sizeof(std::size_t), &nelem, CkReduction::sum_ulong,
-              m_cbp.get< tag::load >() );
+  // Sum number of cells across distributed mesh
+  std::vector< std::size_t > meshdata{ meshid, m_ginpoel.size()/4 };
```

Comment:
Definitely in these cases have meshid set as the refnum. No sense in summing up the meshid everywhere, and it makes the code less understandable.

Reply:
Yeah, I definitely not like that summing either, but I have no idea how to do it cleaner. I will try with refnum.

Reply:
I don't know how I can do this with refnum as the target is not part of an SDAG.

Reply:
I mentioned it above. The other option would be tuple reductions in charm++. But I don't know if I would necessarily call that cleaner. At least not based on the last time I used the tuple interface.

Reply:
Okay, cool. This is okay for now I think.